lib/matplotlib/colors.py

```diff
@@ -1992,3 +1994,49 @@ def from_levels_and_colors(levels, colors, extend='neither'):
 
     norm = BoundaryNorm(levels, ncolors=n_data_colors)
     return cmap, norm
+
+
+def brighten_color(color, frac):
+    """A color helper utility to either darken or brighten the given color.
+
+    This function first converts the given color to RGB using
+    `~.colorConverter` and then to
+    HSL using `colorsys.rgb_to_hsl`.  The lightness is modified according
```

Comment:
Seems foolish to use the non-vectorized `colorsys` functions when we already have a vectorized `rgb_to_hsv` higher up in the file.

I know that HSV and HSL are different, but either we should write a vectorized `rgb_to_hsl` for consistency, or just use the HSV one that's already here

Reply:
This is outside what I'm willing to do for this PR.  

If we could just use HSV, that'd be great, but my understanding is that to get white in HSV you need to change both S and V, whereas in HSL you can just change L.  HSL seems the better colourspace for what we are doing here.  

Just for context, this is an old PR that was sitting around for 2+ years, so I'm willing to put it through, but I'm not interested enough in getting the functionality that I'm willing to write a whole rgb_to_hsl conversion routine.  If someone wants to put in a PR that does that, either before this one is accepted, or after, then that'd be swell.  But I think the PR has merit even if it is not vectorized.